### ISSUE TITLE
errors not displayed

### ISSUE THREAD

**Issue by SpEcHiDe** [maaverik/schemeJS#1]
If there is an error in the syntax, it is not displayed (in some cases!)

**Comment by maaverik** [comment]
Yeah, haven't fixed it completely yet...Gonna do it sometime...Thanks for
the reminder.

**Comment by uc-Pri** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
ArghyadeepHazra, @Dominique-Roth, @enessm, @lealobanov, @joacoforteza, @Alchemyst0x, @mmaurix, @NoNUser2412, @Orkan92, @Rafagil2, @AbhishekRoyalking, @eatmicco, @orbitturner, @Cafen, @Mo-Bhadmus, @HsienChun-Chen, @elpolloj, @IshikuUltra, @tomavanc, @akshatcoder-hash
